login.php

```diff
@@ -61,20 +61,10 @@
 			<div class="logo">
 				<a href = "#"><img src = "images/logo8.png" height = "100"></a>
 			</div>
-			<!--nav for User page -->
-			<nav>
-				<ul>
-					<li><a href = "homepage.php">Home</a></li>
-					<li><a href = "musicplayer.php">Music Play</a>
-						<ul>
-							<li><a href= "Song_sort_Title.php">Title</a></li>
-							<li><a href= "Song_sort_Genre.php">Genre</a></li>
-						</ul>
-					</li>
-					<li><a href = "login.php">LoginOut</a></li>
-					<li><a href = "signup.php">SignUp</a></li>
-				</ul>
-			</nav>
+			<?php
```

Comment:
12/08/2021: I pull nav from the link from nav_login.php. to the login page so it can make it look or clear and easy to understand.